packages/repo/src/util.ts

```diff
@@ -1,9 +1,12 @@
 import { CID } from 'multiformats/cid'
 import * as auth from '@atproto/auth'
+import { def } from '@atproto/common'
 import Repo from './repo'
 import { DataDiff, MST } from './mst'
 import { RepoStorage } from './storage'
 import { DataStore, RecordWriteOp } from './types'
+import BlockMap from './block-map'
+import { CarReader } from '@ipld/car/reader'
```

Comment:
No biggie here, just pulling this dep up since we seem to put external deps at the top of the file by convention.
```suggestion
import { CID } from 'multiformats/cid'
import { CarReader } from '@ipld/car/reader'
import * as auth from '@atproto/auth'
import { def } from '@atproto/common'
import Repo from './repo'
import { DataDiff, MST } from './mst'
import { RepoStorage } from './storage'
import { DataStore, RecordWriteOp } from './types'
import BlockMap from './block-map'
```

Reply:
yup 👍